guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+
+[[Pulp]]
+Pulp:: A service within Katello responsible for repository and content management.
+
+[[Pulp_Mirror]]
+Pulp Mirror:: A {SmartProxyServer} component that mirrors content.
+
+[[Puppet]]
+Puppet:: The configuration management component of {Project}.
+
+[[Puppet_Agent]]
+Puppet Agent:: A service running on a host that applies configuration changes to that host.
+
+[[Puppet_Environment]]
+Puppet Environment:: An isolated set of Puppet Agent nodes that can be associated with a specific set of Puppet Modules.
+
+[[Puppet_Manifest]]
+Puppet Manifest:: Refers to Puppet scripts, which are files with the *.pp* extension.
+The files contain code to define a set of necessary resources, such as packages, services, files, users and groups, and so on, using a set of key-value pairs for their attributes.
++
+Do not confuse with xref:Manifest[Manifest (Red{nbsp}Hat Subscription Manifest)].
+
+[[Puppet_Server]]
+Puppet Server:: A {SmartProxyServer} component that provides Puppet Manifests to hosts for execution by the Puppet Agent.
+
+[[Puppet_Module]]
+Puppet Module:: A self-contained bundle of code (Puppet Manifests) and data (facts) that you can use to manage resources such as users, files, and services.
+
+[[Recurring_Logic]]
+Recurring Logic:: A job executed automatically according to a schedule.
+In the {ProjectWebUI}, you can view those jobs under *Monitor* > *Recurring logics*.
+
+[[Registry]]
+Registry:: An archive of container images.
+{Project} supports importing images from local and external registries.
+{Project} itself can act as an image registry for hosts.
+However, hosts cannot push changes back to the registry.
+
+[[Repository]]
+Repository:: Provides storage for a collection of content.
+
+[[Resource_Type]]
+Resource Type:: Refers to a part of {Project} infrastructure, for example host, {SmartProxy}, or architecture.
+Used in permission filtering.
+
+[[Role]]
+Role:: Specifies a collection of permissions that are applied to a set of resources, such as hosts.
+Roles can be assigned to users and user groups.
+{Project} provides a number of predefined roles.
+
+[[SCAP_content]]
+SCAP content:: A file containing the configuration and security baseline against which hosts are checked.
+Used in compliance policies.
+
+[[Scenario]]
+Scenario:: A set of predefined settings for the {Project} CLI installer.
+Scenario defines the type of installation, for example to install {SmartProxyServer} execute `{installer-scenario-smartproxy}`.
+Every scenario has its own answer file to store the scenario settings.
+
+ifndef::orcharhino,satellite[]
+[[{smart-proxy-context}]]
+{SmartProxy}:: A {SmartProxy} component that can integrate with external services, such as *DNS* or *DHCP*.
+{SmartProxies} are useful for {Project} deployment across various geographical locations.
+endif::[]
+
+[[Standard_Operating_Environment_SOE]]
+Standard Operating Environment (SOE):: A controlled version of the operating system on which applications are deployed.
+
```

Comment:
```suggestion
```
The docs set doesn't actually offer any documentation on building an SOE. And it's just a generic term used in the industry, nothing specific to Foreman.

Reply:
Dropped.